tools/c7n_azure/c7n_azure/provisioning/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+

```

Comment:
What does this empty __init__ file do?

Reply:
It tells python that this directory contains a package (or subpackage) (https://docs.python.org/3/tutorial/modules.html#packages)